portfolio/src/main/webapp/style.css

```diff
@@ -1,5 +1,5 @@
 body {
-    background-color: #937691; /* Light Purple */
+    background-color: #693561; /* Dark Purple */
```

Comment:
If you want a fascinating, if heartbreaking, look into the personal side of web development, learn about the CSS color [rebeccapurple](https://medium.com/@valgaze/the-hidden-purple-memorial-in-your-web-browser-7d84813bb416), which is not too different from the background color you're using here.